### ISSUE TITLE
The new AchorTagHelper in the Media Module needs to be documented

### ISSUE THREAD

**Issue by mbarsott** [OrchardCMS/OrchardCore#7005]
The newly created AnchorTagHelper needs to be documented in order to developers  know of its existence and start using it. 
While at it, we should also make sure the ImgTagHelper is properly documented.

**Comment by Manuel-dev-ing** [comment]
Hi @mbarsott Can I take up this issue?

**Comment by mbarsott** [comment]
Of course! Please go ahead! Thank you!Sent from my iPhoneOn Feb 3, 2026, at 11:00 PM, Manuel Tamayo Montero ***@***.***> wrote:﻿Manuel-dev-ing left a comment (OrchardCMS/OrchardCore#7005)
Hi @mbarsott Can I take up this issue?

**Comment by Manuel-dev-ing** [comment]
Before we start working, I have a question: what structure should the documentation for these Tag Helpers have?

**Comment by mbarsott** [comment]
I’m sorry, I don’t know how to answer that. I have not contributed to the Orchard Core repo in years. Sent from my iPhoneOn Feb 4, 2026, at 9:27 PM, Manuel Tamayo Montero ***@***.***> wrote:﻿Manuel-dev-ing left a comment (OrchardCMS/OrchardCore#7005)
Before we start working, I have a question: what structure should the documentation for these Tag Helpers have?

**Comment by gvkries** [comment]
I believe we don't have a dedicated page for tag helpers. They are documented in the general documentation for the respective module.

**Comment by Manuel-dev-ing** [comment]
I agree with what you mentioned; however, regarding tag helpers, the official .NET documentation already covers them, so creating a separate page for TagHelpers would be redundant. Documentation already exists for the Media module, but only for ImgTagHelper; documentation for AnchorTagHelper is still pending. Therefore, I think the most prudent approach would be to create documentation for AnchorTagHelper following the same structure as the existing documentation.
